tools/VscPreviewExtension/src/contentProvider.ts

```diff
@@ -4,7 +4,7 @@ import { workspace, ExtensionContext, TextDocumentContentProvider, EventEmitter,
 import * as path from "path";
 
 export class ContentProvider implements TextDocumentContentProvider {
-    public port;
+    public static port = "4001";
```

Comment:
no, it should be variable, port is seted 'extension.ts' line 95

---
In reply to: [111559335](https://github.com/dotnet/docfx/pull/1547#discussion_r111559335) [](ancestors = 111559335)